File: crud-jpa-enterprise/crud-jpa-ejb/src/main/java/co/com/foundation/crud/jpa/domain/EmployeeRequest.java
package co.com.foundation.crud.jpa.domain;

import co.com.foundation.crud.jpa.dto.EmployeeDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@Builder
public class EmployeeRequest {

	private boolean update;
	private EmployeeDTO employee;

	public EmployeeRequest() {
		super();
	}
}

File: hr-employees/src/app/mesagges/employeeRequest.ts
import { EmployeeDTO } from "../domain/employeeDTO";

export class EmployeeRequest {

    public update: boolean;
    public employee: EmployeeDTO;

    public constructor() {
        this.init();
    }

    public init(): void {
        this.update = false;
        this.employee = new EmployeeDTO();
    }
}